civic_sandbox/meta.py

```diff
@@ -0,0 +1,91 @@
+
+bikeparking_meta = {
+  'attributes': {
+    'primary': {
+      'field': None,
+      'name': None,
+    },
+    'secondary': {
+      'field': None,
+      'name': None,
+    },
+  },
+    'dates': {
+    'date_column': None,
+    'date_granularity': None,
+    'default_date_filter': '2017',
+    },
+  }
+
+bikelanes_meta = {
+  'attributes': {
+    'primary': {
+      'field': None,
+      'name': None,
+    },
+    'secondary': {
+      'field': None,
+      'name': None,
+    },
+  },
+    'dates': {
+    'date_column': None, 
+    'date_granularity': None,
+    'default_date_filter': '2018',
+    },
+  }
+
+taxlot_block_groups_meta = {
+'attributes': {
+  'primary': {
+    'field': None,
+    'name': None,
+  },
+  'secondary': {
+    'field': None,
+    'name': None,
+  },
+},
+  'dates': {
+  'date_column': 'year',
```

Comment:
Are you attached to "column" here? It would be nice to stay consistent with "attribute".

Reply:
sounds good! will change.